### PR TITLE
Fix build

### PR DESCRIPTION
**Pull request by Thykof**

### CODE REVIEW

File: .github/workflows/build.yml [hunk 1]
```diff
@@ -2,6 +2,7 @@
 on:
   push:
     branches: [ main ]
+  pull_request:
```
Comment (mazmassa) [inline .github/workflows/build.yml:5]: This is a mystery for me... do we really need it!? 🤔

**Review by mazmassa — APPROVED** [review]
LGTM!

File: .github/workflows/build.yml (hunk 1 above)
Comment (Thykof) [reply .github/workflows/build.yml:5]: yes because I want to now if the build work before merging into main branch (this is what broke currently)

Comment (mazmassa) [reply .github/workflows/build.yml:5]: Got it! I see! Thank you! 🙏 

Let me understand better then... shouldn't we specify the types of the PRs to be more selective? 🤔 

```
pull_request:
   types: [opened, reopened, synchronize, ready_for_review, labeled, review_requested, edited]
```

 Omitting that is same to validate for all types? Is that the aim!?

**Review by 0xMazout — APPROVED** [review]
Look like good , i didn't tested it 🦸 
If it's working LGTM

**Pull request merged**